src/pages/help-centre/security.js

```diff
@@ -15,11 +15,13 @@ const ArticleWrapper = styled.div`
     margin-left: 12.6rem;
 `
 
+const StyledHeader = styled(Header)`
+    margin-bottom: 2.4rem;
+`
+
 const VerifyAccount = () => (
     <ArticleWrapper>
-        <Header as="h4" margin=" 0 0 2.4rem 0" lh="1.5">
-            {localize('Do I need to verify my Deriv account?')}
-        </Header>
+        <StyledHeader as="h4">{localize('Do I need to verify my Deriv account?')}</StyledHeader>
```

Comment:
Also here, please check the line-height

Reply:
ive put default header have line-height of 1.5